aries-site/src/layouts/main/Layout.js

```diff
@@ -24,7 +24,7 @@ const filterChildren = (children, type) => {
 export const Layout = ({ children, title }) => {
   // Only expect a single child of the following types
   const mainContent = filterChildren(children, 'MainContent');
-  const sidebar = filterChildren(children, 'SideBar');
+  const selectedNav = 'start';
```

Comment:
@taysea Following our call today, this variable should be determined from the Nav component on the aries-core. 
I think it will be the final step for bringing all the functionality together 🎉